src/main/telemetry/crsf.c

```diff
@@ -59,12 +59,72 @@
 
 #define CRSF_CYCLETIME_US                   100000 // 100ms, 10 Hz
 #define CRSF_DEVICEINFO_VERSION             0x01
-#define CRSF_DEVICEINFO_PARAMETER_COUNT     255
+#define CRSF_DEVICEINFO_PARAMETER_COUNT     0
+
+#define CRSF_MSP_BUFFER_SIZE 96
 
 static bool crsfTelemetryEnabled;
 static bool deviceInfoReplyPending;
 static uint8_t crsfFrame[CRSF_FRAME_SIZE_MAX];
 
+#if defined(USE_MSP_OVER_TELEMETRY)
+typedef struct mspFlatBuffer_s {
+    uint8_t buffer[CRSF_MSP_BUFFER_SIZE];
+    sbuf_t *head; // read-only
+    sbuf_t *tail; // write-only
+} mspFlatBuffer_t;
+
+static mspFlatBuffer_t mspFlatRxBuffer;
+static sbuf_t mspRxBufHead;
+static sbuf_t mspRxBufTail;
```

Comment:
`mspRxBufHead` and `mspRxBufTail` can be declared inside `initCrsfMspBuffer` funtion.